packages/mobile/App.svelte

```diff
@@ -89,21 +102,6 @@
         const platform = await Platform.getOS()
         setPlatform(platform)
     })
-
```

Comment:
Removed fake keyboard code because it was making conflict with the Capacitor keyboard plugin.